docs/docs/cmd/purview/retentionevent/retentionevent-list.md

```diff
@@ -0,0 +1,100 @@
+# purview retentionevent list
+
+Get a list of retention events
+
+## Usage
+
+```sh
+m365 purview retentionevent list [options]
+```
+
+## Options
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Get a list of retention events
+
+```sh
+m365 purview retentionevent list
+```
+
+## Remarks
+
+!!! attention
+    This command is based on a Microsoft Graph API that is currently in preview and is subject to change once the API reached general availability.
+
+## Response
+
+
+=== "JSON"
+
+    ```json
+    [
+      {
+        "displayName": "Retention Event",
```

Comment:
Could you give this a meaningful displayName. For example: 'Contract xyz expired' (this maps to event type 'Contract Expiry Event', which we use in the other commands as an example)